contrib/labstack/echo.v4/echotrace.go

```diff
@@ -52,11 +52,7 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 		return func(c echo.Context) error {
 			// If we have an ignoreRequestFunc, use it to see if we proceed with tracing
 			if cfg.ignoreRequestFunc != nil && cfg.ignoreRequestFunc(c) {
-				if err := next(c); err != nil {
-					c.Error(err)
-					return err
-				}
-				return nil
+				return next(c)
```

Comment:
I think we should add some test to verify this behavior and ensure we don't introduce a regression in the future.

Reply:
@rarguelloF Actually, if somebody was relying in the existing behaviour, they probably won't feel a difference. The existing behaviour blocked middlewares running before our own to manipulate the response, as `c.Error` already commited it to the client.

I think it's safe to go ahead as it is right now.